README.md

```diff
@@ -2700,12 +2700,18 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 - <a name="Joubert21fins"></a>Joubert, C., Marcireau, A., Ralph, N., Jolley, A., van Schaik, A., Cohen, G.,  
 [Event Camera Simulator Improvements via Characterized Parameters](https://www.frontiersin.org/articles/10.3389/fnins.2021.702765/full)  
   Front. Neurosci., 2021, [IEBCS simulator](https://github.com/neuromorphicsystems/IEBCS)
+- <a name="Gu24mm"></a>Gu, D., Li, J., Zhang, Y., Tian, Y.,   
+*[How to Learn a Domain-Adaptive Event Simulator](http://dx.doi.org/10.1109/TPAMI.2023.3326538)*,  
+ACM Int. Conf. on Multimedia (MM), 2021. [PDF](https://doi.org/10.1145/3474085.3475229), [Code](https://github.com/iCVTEAM/LETGAN).
```

Comment:
No need to have [PDF](https://doi.org/10.1145/3474085.3475229) if it is the same link as in the title of the paper.
This [PDF] is intended to provide a pdf that is not behind a paywall.

Reply:
Thanks. I have linked it to a PDF that not behind a paywall.